hazelcast-jet-core/src/test/java/com/hazelcast/jet/core/CancellationTest.java

```diff
@@ -297,6 +297,7 @@ private static void assertExecutionTerminated() {
                 long last = previous[0];
                 previous[0] = current;
                 assertTrue("Call counter should eventually stop being incremented.", current == last);
+                sleepMillis(200);
```

Comment:
I don't think we need a `sleep` call here. `assertTrueEventually()` already sleeps internally.

Reply:
That's true, but it's not part of its contract. We require longer pauses for correct functionality, so why not do them ourselves?